storage/block/how-to/increase-block-volume.mdx

```diff
@@ -36,6 +38,66 @@ With Block Storage it is possible to expand the volume size of an instance.
  
   </Message>
 
+## How to increase the parition size of the volume
+
+<Message type="tip">
+
+It is recommeneded to make a backup of your data using the [Snapshot](https://www.scaleway.com/en/docs/compute/instances/how-to/create-a-snapshot/) feature, before increasing the partition size of your volume to avoid any data loss. 
```

Comment:
```suggestion
It is recommeneded to make a backup of your data using the [Snapshot](https://www.scaleway.com/en/docs/compute/instances/how-to/create-a-snapshot/) feature, before increasing the partition size of your volume. This helps to avoid any data loss. 
```